### PR TITLE
Update DB on pkg remove

### PR DESCRIPTION
**Pull request by dsimog01**
On pkg remove, the DB should be updated if that pkg was selected in the stakers tab

### CODE REVIEW

**Comment by 3alpha** [comment]
Is there reason no to handle that here: https://github.com/dappnode/DNP_DAPPMANAGER/blob/develop/packages/dappmanager/src/calls/packageRemove.ts?

File: packages/dappmanager/src/daemons/stakerDbUpdate/index.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { mevBoostMainnet, mevBoostPrater, stakerPkgs } from "@dappnode/types";
+import * as db from "../../db/index.js";
+import { logger } from "ethers";
+
+async function removeStakerPkgsFromDbIfSelected({
+  dnpNames
+}: {
+  dnpNames?: string[];
+}): Promise<void> {
+  if (!dnpNames) return;
```
Comment (pablomendezroyo) [inline packages/dappmanager/src/daemons/stakerDbUpdate/index.ts:11]: this checker will not work as you expect. The `if(!...)` does not check for arrays with length 0. You must check for `if(dnpNames.length === 0)

File: packages/dappmanager/src/daemons/stakerDbUpdate/index.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        break;
+      case "mev-boost-goerli.dnp.dappnode.eth":
+        db.mevBoostPrater.set(false);
+        break;
+      default:
+        break;
+    }
+  });
+
+  logger.info(`Removed clients/mev-boost ${dnpNames} from main DB`);
```
Comment (pablomendezroyo) [inline packages/dappmanager/src/daemons/stakerDbUpdate/index.ts:43]: Too ambiguous and not precise. Not all packages might belong to the main DB. Either remove it or improve it

File: packages/dappmanager/src/daemons/stakerDbUpdate/index.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      case db.consensusClientMainnet.get():
+        db.consensusClientMainnet.set(undefined);
+        break;
+      case db.consensusClientGnosis.get():
+        db.consensusClientGnosis.set(undefined);
+        break;
+      case db.consensusClientPrater.get():
+        db.consensusClientPrater.set(undefined);
+        break;
+      case "mev-boost.dnp.dappnode.eth":
```
Comment (pablomendezroyo) [inline packages/dappmanager/src/daemons/stakerDbUpdate/index.ts:32]: Use the dnpName provided by the library @dappnode/types

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

**Pull request merged**